presto-hive/src/test/java/com/facebook/presto/hive/benchmark/FileFormat.java

```diff
@@ -599,6 +599,7 @@ public PrestoOrcFormatWriter(File targetFile, List<String> columnNames, List<Typ
                     ImmutableMap.of(),
                     hiveStorageTimeZone,
                     false,
+                    false,
```

Comment:
What is the reason for this additional parameter ?

Reply:
Removed - not required with the new flow.